app/templates/components/forms/session-speaker-form.hbs

```diff
@@ -332,9 +332,9 @@
       Or
     </div>
     <div class="field">
-      {{ui-radio name='sessionDetails' label=(t 'Add a new Proposal') value=false onChange=(action 'toggleNewSessionSelected' true)}}
+      {{ui-checkbox name='sessionDetails' class="toggle" label="Add a new proposal" checked=addNewSession onChange=(action (mut addNewSession))}}
```

Comment:
Why renaming the actons ?

Reply:
if i will use (mut addNewsession) there will be no need of 
toggleNewSessionSelected(value) {
      this.set('sessionDetails', false);
      this.set('newSessionSelected', value);
    }
in session-form.js . So can i remove this action from session-speaker-form.js

Reply:
And also same action is not working correctly - 
![captured (3)](https://user-images.githubusercontent.com/56407566/74084883-98122880-4a99-11ea-9b7c-7cade883c66e.gif)

Reply:
Use class ui-checkbox with old actions. It should work .

Reply:
It's not working too - 
![captured](https://user-images.githubusercontent.com/56407566/74156544-887f1500-4c3c-11ea-8630-c85d74cac779.gif)